teamwork/south_migrations/0001_initial.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
```

Comment:
Oh, never mind! Looks like `south_migrations` is [a documented alternate directory](https://docs.djangoproject.com/en/dev/topics/migrations/#libraries-third-party-apps) for this exact purpose.